pkg/i18n/english.go

```diff
@@ -1182,6 +1184,8 @@ func EnglishTranslationSet() *TranslationSet {
 		ForceCheckoutTooltip:                 "Force checkout selected branch. This will discard all local changes in your working directory before checking out the selected branch.",
 		CheckoutByName:                       "Checkout by name",
 		CheckoutByNameTooltip:                "Checkout by name. In the input box you can enter '-' to switch to the last branch.",
+		CheckoutLastBranch:                   "Checkout last branch",
+		CheckoutLastBranchTooltip:            "Checkout to the last branch.",
```

Comment:
I don't find this tooltip very valuable when it adds almost nothing beyond what's already in the name of the command. I'd either drop it, or make it more elaborate, e.g. "Checkout the previously checked out branch again" or something like that.

Reply:
I agree.
I just added tooltip description somehow. I'll delete it.